tests/test_gestalt.py

```diff
@@ -471,8 +470,27 @@ def test_vault_get():
     client_config['verify'] = True
     g.add_vault_config_provider(client_config, auth_config=None)
     print("Requires the user to set a token in the client")
-    CLIENT_ID = "test_client"
-    g.add_vault_secret_path("test")
+    client_id = "test_client"
+    client_password = "test_client_password"
+    g.add_vault_secret_path(path="test")
+    g.fetch_vault_secrets()
+    secret = g.get_string(client_id)
+    assert secret == client_password
+
+
+def test_vault_mount_path():
+    g = gestalt.Gestalt()
+    g.build_config()
+    client_config = gestalt.HVAC_ClientConfig()
+    client_config['url'] = ""
+    client_config['token'] = "myroot"
+    client_config['cert'] = None
+    client_config['verify'] = True
+    g.add_vault_config_provider(client_config, auth_config=None)
+    print("Requires the user to set a token in the client")
+    client_id_mount_path = "test_mount"
```

Comment:
might want to make this `test_path` to avoid confusion for the next person

Reply:
this is just a value, which is why `client_id_mount_path` is used everywhere. We don't work with `test_mount` furthermore than this step. :)